utils/utils.go

```diff
@@ -97,14 +99,20 @@ func GetRevision() string {
 }
 
 func init() {
-	ViamDirs.Viam = "/opt/viam"
-	if runtime.GOOS == "windows" {
-		ViamDirs.Viam = "c:/opt/viam"
-		// note: forward slash isn't an abs path on windows, but resolves to one.
-		var err error
-		ViamDirs.Viam, err = filepath.Abs(ViamDirs.Viam)
-		if err != nil {
-			panic(err)
+	ReinitViamDirs()
+}
+
+func ReinitViamDirs() {
+	if ViamDirs.Viam == "" {
```

Comment:
Bit of a nitpick but can we parameterize the value for `ViamDirs.Viam` here? I find needing to manipulate some global state before calling the function [like this][1] pretty unintuitive.  

```suggestion
func ReinitViamDirs(viamDir string) {
    ViamDirs.Viam = viamDir
	if ViamDirs.Viam == "" {
```

[1]: https://github.com/aldenh-viam/agent/blob/ca3a916d686bda91434947d7412cc597cc23cb5a/cmd/viam-agent/main.go#L111-L112